README.md

```diff
@@ -6,21 +6,63 @@
 
 fleet-server is under development. The following are notes to help developers onboarding to the project to quickly get running. These notes might change at any time.
 
-### Startup fleet-server
+## Setup
 
-Currently to startup fleet-server, the Kibana encryption key is needed. There are two options for this.
+To run and test fleet-server, a recent version of Elastic Agent and Kibana are needed. In the following Elastic Agent and Kibana are built from source. The fleet-server itself is not built from source but pulled from the latest snapshot build. It would be possible to also pull Elastic Agent or Kibana from the latest snapshot but the assumption that is made here that whoever is testing this, is likely developing either Elastic Agent or on the Kibana side.
```

Comment:
If the expectation is that you are developing with Elastic Agent or Kibana, why not also the Fleet Server? I think we should make it consistent and have this include how to build Fleet Server.

It really is as simple as `make release` inside of the Fleet Server repository.

Reply:
`make release` and then copy it over to the right place I assume?

Reply:
Yes, it needs to be copied.

Reply:
@blakerouse Would you be ok to get it in as is and follow up with the fleet-server part?

Reply:
Sure!